.github/pull_request_template.md

```diff
@@ -1,23 +1,27 @@
-## [Replace With Ticket Number]
+---
+name: Pull Request
+about: Describe the purpose and details of your PR
+---
 
-<!-- Provide a brief description of the changes made in this PR -->
+## [NEW-XXXX] Summary
+<!-- Provide a brief explanation of the changes -->
 
-## Testing Steps
 
-<!-- Provide testing steps and reference storybook stories if necessary -->
-<!-- Add applicable configs to JIRA ticket for testers-->
+## How did you test this change?
```

Comment:
I like Testing Steps better. I want to get easy instructions for booting up the change and seeing the change rather than some explanation.

Reply:
Agreed - removed required checks on the summary section.

Reply:
no like I want this to say "Testing Steps" again

Reply:
Replaced the text with "Testing Steps" and updated the job that checks for that text.